docs/diagrams/wallet-lifecycle/wallet-lifecycle-5-handling-moved-funds.tikz

```diff
@@ -0,0 +1,129 @@
+% TIP: To generate `.pdf` file out of this `.tikz` file run:
+% `pdflatex filename.tikz`.
+% To install `pdflatex` on MacOS either run `brew install --cask mactex` (not
+% recommended, downloads 4GB of data) or install `basictex` distribution plus
+% needed packages (recommended, downloads only essential packages):
+% ```
+% brew install --cask basictex
+% sudo tlmgr update --self
+% sudo tlmgr install standalone
+% sudo tlmgr install comment
+% pdflatex <filename>.tikz
+%```
+% 
+% To generate `.png` file out of `.pdf` on MacOS run:
+% `convert -density 300 <filename>.pdf -resize 50% <filename>.png`.
+% This requires installation of the `imagemagick` tool:
+% `brew install imagemagick`.
+
+\documentclass{standalone}%
+\usepackage{tikz}%
+\usetikzlibrary{positioning,shapes,arrows.meta,chains}%
+
+% These commands are used to draw arrow labels
+\newcommand{\yes}{node [auto, pos=0, xshift=5mm]{Y}}%
+\newcommand{\no}{node [auto, swap, pos=0, xshift=-5mm]{N}}%
+
+% We need layers to draw the background
+\pgfdeclarelayer{background}%
+\pgfsetlayers{background,main}%
+
+\begin{document}%
+\begin{tikzpicture}[
+% Specify styles for nodes:
+    base/.style = {rectangle, rounded corners, draw=black,
+                   minimum width=4cm, minimum height=1cm,
+                   text centered, text width=4cm, align=flush center},
+   start/.style = {base, fill=blue!30},
+  revert/.style = {base, fill=red!30},
+  action/.style = {base, fill=black!5},
+   check/.style = {diamond, aspect=1.2, draw=black, fill=black!5,
+                   minimum width=1cm, align=center},
+   note1/.style = {rectangle, draw, dashed, fill=green!10,
+                   align=flush left, text width=5cm},
+   note2/.style = {rectangle, draw, dashed, fill=cyan!10,
+                   align=flush left, text width=5cm},
+              > = Triangle,                  % defines the arrow tip style when `->` is used
+           font = \sffamily,
+]
+
+% MOVED FUNDS SWEEP PROOF
+% Nodes
+\node[start]                                                       (mfsp_start)   {Somebody sends proof of moved funds sweep};
+\node[check, below=of mfsp_start]                                  (mfsp_check1)  {Is there\\ a pending moved\\ funds sweep\\ request registered\\ for the wallet?};
+\node[check, below right=of mfsp_check1]                           (mfsp_check2)  {Has the\\ sweep transaction\\ occurred on\\ Bitcoin?};
+\node[check, below right=of mfsp_check2]                           (mfsp_check3)  {Does the\\ sweep transaction have\\ a single output?};
```

Comment:
Not in this PR, but we can improve this diagram by adding one more condition node (or two?):

If the wallet has main UTXO set, we expect two inputs in the sweep transaction.
If the wallet has no main UTXO set, we expect one input in the sweep transaction.

Reply:
Applied in https://github.com/keep-network/tbtc-v2/pull/335.